ansible/deploy_netvault.yml

```diff
@@ -1,11 +1,11 @@
 - name: "Install Netvault Client"
-  hosts: pg_standby:couchdb2
+  hosts: pg_backup
```

Comment:
pg_backup doens't really have a running instance of postgresql so pg_dump won't work there, and netvault uses pg_dump, so I am not sure this will work, unless we try offline backups with netvault.

Reply:
Yeah, it was a wrong update (used for local test). I have updated it now